content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,487 @@
+        }
+      }
+    }
+  ],
+  "meta": {
+    "page": {
+      "total": 0,
+      "page_number": 0,
+      "page_size": 0
+    }
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Get a specific AuthN Mapping
+
+Returns a specific AuthN Mapping by UUID.
+
+| Method | Endpoint path | Required payload                           |
+|--------|--------------|--------------------------------------------|
+| `GET` | `/authn_mappings/{UUID}`  | URL parameter |
+
+##### ARGUMENTS
+
+* **`{UUID}`** [*required*, no default]:
+  Replace `{UUID}` with the UUID of the AuthN Mapping you want to view.
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+```sh
+curl -X GET "https://app.datadoghq.com/api/v2/authn_mappings/{UUID}" \
+     -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+     -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>"
+```
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholders with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+```json
+{
+  "data": {
+    "attributes": {
+      "created_at": "string",
+      "created_by": "string",
+      "uuid": "10000000-abcd-efgh-ijkl-000000000000",
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "saml_assertion_attribute_id": 0
+    },
+    "type": "string",
+    "id": "string",
+    "relationships": {
+      "saml_assertion_attributes": {
+        "data": {
+          "id": 0,
+          "type": "saml_assertion_attributes"
+        }
+      },
+      "roles": {
+        "data": {
+          "id": "string",
+          "type": "roles"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "data": {
+        "id": "string",
+        "type": "roles",
+        "attributes": {
+          "created_at": "string",
+          "modified_at": "string",
+          "uuid": "00000000-0000-0000-0000-000000000000",
+          "name": "string"
+        },
+        "relationships": {
+          "data": [
+            {
+              "id": "string",
+              "type": "permissions"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "data": {
+        "id": 0,
+        "type": "authn_mappings",
+        "attributes": {
+          "id": 6,
+          "attribute_key": "string",
+          "attribute_value": "string"
+        }
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Update mapping
+
+Updates the AuthN Mapping `role`, `saml_assertion_attribute_id`, or both from a JSON body. Returns the updated AuthN Mapping.
+
+
+| Method | Endpoint path | Required payload |
+|--------|--------------|------------------|
+| `PATCH`  | `/v2/authn_mappings/{UUID}`  | JSON       |
+
+##### ARGUMENTS
+
+* **`{UUID}`** [*required*, no default]:
+  Replace `{UUID}` with the UUID of the AuthN Mapping you want to update.
+* **`role_uuid`** [*required*, *default*=**?**]:
+?
+* **`attribute_key`** [*required*, *default*=**?**]:
+?
```

Comment:
Defaults for update should be None

Some context for this:
If a user of the API passes a role_uuid we’ll update the role, if a user passes in BOTH an attribute_key and attribute_value, we’ll update the underlying saml_assertion_attribute, if you pass in only a PART of a saml_assertion_attribute, we will not update the saml_assertion_atttribute portion, if you pass in BOTH a role_uuid and an attribute key + attribute_value, we will update the underlying role and saml_assertion_attribute